app/controllers/organizations_controller.rb

```diff
@@ -5,6 +5,11 @@ class OrganizationsController < ApplicationController
   def index
   end
 
+  def new
+    raise PermissionError unless current_user.can_create_organization?
```

Comment:
They won't have come from anywhere. You can't get to this page unless you have permission.